soco/music_services/accounts.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+
+"""Accounts for Third Party music services."""
+
+from __future__ import unicode_literals
+
+import weakref
+import logging
+log = logging.getLogger(__name__)  # pylint: disable=C0103
+
+import requests
+
+from soco.core import SoCo
```

Comment:
Yes, good idea.  We should use from __future__ import absolute_imports everywhere, and then use relative imports for solo.  I'll change this ref.